cve_bin_tool/checkers/__init__.py

```diff
@@ -22,4 +22,5 @@
     "xml2",
     "zlib",
     "python",
+    "jetty",
```

Comment:
Jetty?

Reply:
Sorry for the mess up, was working on another issue, will do the required changes.